Sources/ATTNCustomEvent.m

```diff
@@ -18,7 +18,6 @@ - (instancetype)initWithType:(NSString *)type properties:(NSDictionary<NSString
     NSString *invalidCharInType = [ATTNCustomEvent findInvalidCharacterInType:type];
     if (invalidCharInType != nil) {
       NSLog(@"Invalid character '%@' in CustomEvent type '%@'", invalidCharInType, type);
-      [NSException raise:@"Bad CustomEvent Type" format:@"The CustomEvent type '%@' had an invalid character '%@'.", type, invalidCharInType];
```

Comment:
I think if we do encounter an exception like this here and below, we'd want to not create the event & return null or something